//[validator-field](../../../index.md)/[com.chrynan.validator](../index.md)/[FieldValidationException](index.md)

# FieldValidationException

[common]\
class [FieldValidationException](index.md) : [RuntimeException](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/-runtime-exception/index.html)

A [RuntimeException](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/-runtime-exception/index.html) that indicates there was an error asserting that a field was valid or invalid.

This exception is thrown from the [assertValidField](../assert-valid-field.md) and [assertInvalidField](../assert-invalid-field.md) functions.

## Properties

| Name | Summary |
|---|---|
| [cause](index.md#-654012527%2FProperties%2F-155168089) | [common]<br>open val [cause](index.md#-654012527%2FProperties%2F-155168089): [Throwable](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/-throwable/index.html)? |
| [message](index.md#1824300659%2FProperties%2F-155168089) | [common]<br>open val [message](index.md#1824300659%2FProperties%2F-155168089): [String](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/-string/index.html)? |
